spec/ataru/hakija/validator_spec.clj

```diff
@@ -282,62 +298,94 @@
-      (should (:passed? (validator/valid-application?
+      (should (:passed? (validator/valid-application? has-never-applied
                          a
                          (update f :content conj hakukohde-specific-dropdown-with-followups)))))
 
   (it "passes validation when a dropdown question is hakukohde specific and has answers",
-      (should (:passed? (validator/valid-application?
+      (should (:passed? (validator/valid-application? has-never-applied
                          (update a :answers conj hakukohde-answer dropdown-answer dropdown-followup-answer)
                          (update f :content conj hakukohde-question hakukohde-specific-dropdown-with-followups)))))
 
   (it "fails validation when a dropdown question is hakukohde specific and has no required followup answers",
-      (should-not (:passed? (validator/valid-application?
+      (should-not (:passed? (validator/valid-application? has-never-applied
                              (update a :answers conj hakukohde-answer dropdown-answer)
                              (update f :content conj hakukohde-question hakukohde-specific-dropdown-with-followups)))))
 
   (it "passes validation when a dropdown question is hakukohde specific to wrong hakukohde and has no answers",
-      (should (:passed? (validator/valid-application?
+      (should (:passed? (validator/valid-application? has-never-applied
                          (update a :answers conj hakukohde-answer)
                          (update f :content conj hakukohde-question (assoc hakukohde-specific-dropdown-with-followups
                                                                           :belongs-to-hakukohteet
                                                                           ["1.2.246.562.20.352373851711"]))))))
 
   (it "fails validation when a dropdown question is hakukohde specific to wrong hakukohde and has answers",
-      (should-not (:passed? (validator/valid-application?
+      (should-not (:passed? (validator/valid-application? has-never-applied
                              (update a :answers conj hakukohde-answer dropdown-answer dropdown-followup-answer)
                              (update f :content conj hakukohde-question (assoc hakukohde-specific-dropdown-with-followups
                                                                                :belongs-to-hakukohteet
-                                                                               ["1.2.246.562.20.352373851711"])))))))
+                                                                               ["1.2.246.562.20.352373851711"]))))))
+
+  (it "fails validation when cannot submit multiple applications and has applied"
+      (let [has-applied (fn [_ _] (async/go true))
+            form (populate-can-submit-multiple-applications f false)
+            answers (update a :answers (partial remove #(= "birth-date" (:key %))))]
+        (should-not (:passed? (validator/valid-application?
```

Comment:
This ship has probably sailed with old tests, but with new ones i'd love to see the assert on _why_ the validation (saving etc.) has failed. Just a suggestion though..